ServiceMetadata class missing but used by wmts and tms

I was just fixing the pep8 issues (#452) after updating to `flake8`.  `flake8` is complaining about unknown `ServiceMetadata` in `wmts` module (same for `tms`):
```
$ flake8 owslib/wmts.py 
owslib/wmts.py:195:34: F821 undefined name 'ServiceMetadata'
```

See code:
https://github.com/geopython/OWSLib/blob/c5afe82324c5ffe2b6aed34e59babbb40c13df3b/owslib/wmts.py#L195

I couldn't find a `ServiceMetadata` in `owslib`.  Was it renamed, removed or does it come from an external library? After a short research it looks like `ServiceMetadata` is missing for a few years.

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,7 +14,7 @@ install:
   - if [ "$LXML" == "true" ]; then pip install lxml; fi
 script:
   - python -m pytest
-  #- flake8 owslib/wmts.py
+  - flake8 owslib/wmts.py
   - flake8 owslib/wps.py
 after_success:
   - coveralls

--- a/owslib/tms.py
+++ b/owslib/tms.py
@@ -65,11 +65,13 @@ class TileMapService(object):
 
 
     def _getcapproperty(self):
+        # TODO: deprecated function. See ticket #453.
         if not self._capabilities:
             reader = TMSCapabilitiesReader(
                 self.version, url=self.url, un=self.username, pw=self.password
                 )
-            self._capabilities = ServiceMetadata(reader.read(self.url))
+            # self._capabilities = ServiceMetadata(reader.read(self.url))
+            self._capabilities = reader.read(self.url, timeout=self.timeout)
         return self._capabilities
 
 

--- a/owslib/wmts.py
+++ b/owslib/wmts.py
@@ -187,12 +187,14 @@ class WebMapTileService(object):
         self._buildMetadata(parse_remote_metadata)
 
     def _getcapproperty(self):
+        # TODO: deprecated function. See ticket #453.
         if not self._capabilities:
             reader = WMTSCapabilitiesReader(
                 self.version, url=self.url, un=self.username, pw=self.password
             )
-            xml = reader.read(self.url, self.vendor_kwargs)
-            self._capabilities = ServiceMetadata(xml)
+            # xml = reader.read(self.url, self.vendor_kwargs)
+            # self._capabilities = ServiceMetadata(xml)
+            self._capabilities = reader.read(self.url, self.vendor_kwargs)
         return self._capabilities
 
     def _buildMetadata(self, parse_remote_metadata=False):

